resources/public/api/conf/2.0/docs/overview.md

```diff
@@ -1,45 +1,48 @@
+1. Register for a [developer account](/developer/registration).
 
-2. Create an application.    
+2. Create an application. 
 
-3. Subscribe to any API you might be working on. We suggest [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0) along with [Common Transit Convention Traders API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/common-transit-convention-traders/1.0) and Common Transit Convention Traders Test Support API (which is on this page).     
+3. Subscribe to any API you might be working on. We suggest the [Create Test User API](/api-documentation/docs/api/service/api-platform-test-user/1.0), the [CTC Traders API](/api-documentation/docs/api/service/common-transit-convention-traders/1.0) and this API.
 
-4. Create a [Client ID and Client Secret](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/credentials).
+4. Create a [Client ID and Client Secret](/api-documentation/docs/authorisation/credentials).
 
-5. Use the [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0) to get a user ID, password, EORI enrolment and other test details. 
+5. Use the [Create Test User API](/api-documentation/docs/api/service/api-platform-test-user/1.0) to get a user ID, password, EORI enrolment and other test details. 
 
-<!--For further details on how to get set up for testing, see [CTC Traders API testing guide](/guides/ctc-traders-phase5-testing-guide).-->
+<!--For more information about how to get set up for testing, see [CTC Traders API Testing guide](/guides/ctc-traders-phase5-testing-guide).-->
 
-## How to use the CTC Trader Test Support API
+## How to use this API
 
-You can use this API for any test scenario you choose. It is not connected to the NCTS core. It’s up to you to trigger the response messages in the order you require.  Valid response message types are listed below.
+This API is not connected to the NCTS core, so you can use it for any test scenario you choose.
 
-1.  Send messages from a trader to NCTS using the [CTC Traders API](/api-documentation/docs/api/service/common-transit-convention-traders/2.0).  
+It’s up to you to trigger the response messages in the order you require. Valid response message types are listed below.
 
-2. Inject a response using the Test Support API. See the following table for a list of supported messages you can inject. 
+Follow these steps:
 
-3. GET a list of your messages for this movement from the CTC Traders API. This will include both your trader and your injected NCTS messages.
+1. Use the [CTC Traders API](/api-documentation/docs/api/service/common-transit-convention-traders/2.0) to send messages from a trader to the NCTS.
 
-<br></br>
+2. Use this API to inject a response using this API. See the table below for a list of supported messages that you can inject. 
```

Comment:
How about just "Use this API to inject a response."

Reply:
Well spotted. Missed that. Will fix.